Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "gdbm"
-version = "0.1.1"
+version = "0.1.2"
 authors = ["Chris Holcombe <xfactor973@gmail.com>"]
 homepage = "https://github.com/cholcombe973/gdbm"
 description = "Safe wrapper for gdbm-sys bindings"
 license = "MIT"
 
 [dependencies]
-bitflags = "~0.8"
-gdbm-sys = "~0.2"
+bitflags = "~1.2"
+gdbm-sys = { version = "~0.3", git = "https://github.com/miquels/gdbm-sys" }
```

Comment:
Since we merged the change for gdbm-sys do you want to update this now? If not I can do it later after the merge.

Reply:
Oh yes, sure. I'll increment the version to 0.2.0 as well.